aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/AWSMobileClient.java

```diff
@@ -2453,25 +2566,29 @@ public void run() {
                     }
                 }
 
-                userpool.getCurrentUser().updateAttributes(cognitoUserAttributes, new UpdateAttributesHandler() {
-                    @Override
-                    public void onSuccess(List<CognitoUserCodeDeliveryDetails> attributesVerificationList) {
-                        final List<UserCodeDeliveryDetails> list = new LinkedList<UserCodeDeliveryDetails>();
-                        for (CognitoUserCodeDeliveryDetails details : attributesVerificationList) {
-                            list.add(new UserCodeDeliveryDetails(
-                                    details.getDestination(),
-                                    details.getDeliveryMedium(),
-                                    details.getAttributeName()
-                            ));
+                userpool.getCurrentUser().updateAttributes(
+                        cognitoUserAttributes,
+                        clientMetadata,
+                        new UpdateAttributesHandler() {
+                        @Override
```

Comment:
indentation feels weird here?

Reply:
Good call - yeah it needed an extra indent